spec/requests/schools/participants_request_spec.rb

```diff
@@ -67,13 +69,13 @@
 
   describe "GET /schools/:school_id/cohorts/:start_year/participants/:id" do
     it "renders participant template" do
-      get "/schools/#{school.slug}/cohorts/#{cohort.start_year}/participants/#{ect_user.id}"
+      get "/schools/#{school.slug}/cohorts/#{cohort.start_year}/participants/#{ect_profile.id}"
```

Comment:
I guess we can hope that people don't bookmark those urls, and it's probably a fair bet.

Reply:
Yeah I'm not too worried